src/test/java/com/jcabi/aether/RootArtifactTest.java

```diff
@@ -86,6 +91,32 @@ public void resolvesRootArtifact() throws Exception {
     }
 
     /**
+     * RootArtifact can resolve a parent artifact.
+     * @throws Exception If there is some problem inside
+     */
+    @Ignore
+    @Test
+    public void resolvesParent() throws Exception {
+        final RootArtifact root = new RootArtifact(
+            this.aether(),
+            new DefaultArtifact(
+                "com.jcabi", "jcabi-aether", "", "jar", "0.10.1"
```

Comment:
@prondzyn I think we should those strings that are used repeatedly with meaning (even as part of another string, like the `"0.10.1"` here) into variables and use `String.format()` [here](https://github.com/jcabi/jcabi-aether/pull/91/files#diff-47dd39ca80fdd996f47528d633416157R110)